benchmark-caliper/build.gradle

```diff
@@ -0,0 +1,142 @@
+buildscript {
```

Comment:
Good point, Android is always Caliper.  Folded the stuff into benchmark-android.